pygmo/docstrings.cpp

```diff
@@ -87,11 +87,11 @@ If the problem is multi-objective one single best is not well defined. In this c
 individuals by calling the :func:`pygmo.sort_population_mo()` function.
 
 Args:
-    tol (``float`` or array-like object): scalar tolerance or vector of tolerances to be applied to each constraints. By default, the
+    tol (:class:`float` or array-like object): scalar tolerance or vector of tolerances to be applied to each constraints. By default, the
       :attr:`~pygmo.problem.c_tol` attribute from the population's problem is used.
 
 Returns:
-    ``int``: the index of the best individual
+    :class:`float`: the index of the best individual
```

Comment:
Looks like something went wrong with the regexp. This should read ":class:\`int\`". Same in a bunch of occurrences below.